test/unit/api.js

```diff
@@ -0,0 +1,159 @@
+define([
+    'test/underscore',
+    'jquery',
+    'api/api',
+    'utils/backbone.events'
+], function (_, $, Api, Events) {
+    /* jshint qunit: true */
+
+    module('Api');
+
+    test('extends Events', function(assert) {
```

Comment:
The test passes whether we extend or not, but it will verify for us that the methods actually exist. I propose we change this to "contains expected methods" and test it for these
```
setup
load
play
pause
seek
stop
remove
resize
playlistItem
playlistNext
playlistPrev
registerPlugin
addButton
removeButton
get
set
on
off
once
trigger
```